### ISSUE TITLE
Buffalo Plugins documentation is outdated

### ISSUE THREAD

**Issue by tatang26** [gobuffalo/docs#680]
As title says, this doc topic is outdated due the following:

- It still indicating to install a deprecated library: `github.com/gobuffalo/buffalo-plugins` 
- `Writing a Plugin` section are not clear. pe,`$ buffalo generate plugin github.com/foo/buffalo-bar` doesn't work and is needed to update the content of generated files

**Comment by sio4** [comment]
Agree! I guess it could be/should be updated once the progress on CLI by @paganotoni is done.
